application/src/ext/java/org/opentripplanner/ext/carpooling/model/CarpoolTrip.java

```diff
@@ -40,7 +42,7 @@
  * <p>
  * CarpoolTrip instances are immutable. Updates to trip state (e.g., adding a booked passenger)
  * require creating a new trip instance via {@link CarpoolTripBuilder} and upserting it to the
- * {@link org.opentripplanner.ext.carpooling.CarpoolingRepository}.
+ * {@link CarpoolingRepository}.
```

Comment:
I made a comment about usage of a full path instead of importing a class in code, but for javadoc, you should use the full path instead of importing when the referenced class isn't imported for other reasons. You made the same change in some other places as well in the recent commit.